src/neo/VM/Helper.cs

```diff
@@ -46,36 +48,41 @@ public static ScriptBuilder Emit(this ScriptBuilder sb, params OpCode[] ops)
             return sb;
         }
 
-        public static ScriptBuilder EmitDynamicCall(this ScriptBuilder sb, UInt160 scriptHash, string operation)
+        public static ScriptBuilder EmitDynamicCall(this ScriptBuilder sb, UInt160 scriptHash, string operation, params object[] args)
         {
-            sb.Emit(OpCode.NEWARRAY0);
-            sb.EmitPush(CallFlags.All);
-            sb.EmitPush(operation);
-            sb.EmitPush(scriptHash);
-            sb.EmitSysCall(ApplicationEngine.System_Contract_Call);
-            return sb;
+            return EmitDynamicCall(sb, null, scriptHash, operation, args);
         }
 
-        public static ScriptBuilder EmitDynamicCall(this ScriptBuilder sb, UInt160 scriptHash, string operation, params ContractParameter[] args)
+        public static ScriptBuilder EmitDynamicCall(this ScriptBuilder sb, DataCache snapshot, UInt160 scriptHash, string operation, params object[] args)
```

Comment:
Is it better to add `CallFlags` as the parameter?

Reply:
optional or two methods? I prefer two methods because the `params`

Reply:
Two overloads.

Reply:
Done